README.md

```diff
@@ -76,6 +76,7 @@ OPTIONS:
     -g, --gid               Group id to run with
     -s, --signal            Signal to send to the command when exit it (default: 1, SIGHUP)
     -a, --url-arg           Allow client to send command line arguments in URL (eg: http://localhost:7681?arg=foo&arg=bar)
+    -f, --arg-file          File prefix for a unique generated temp file that URL arguments are written to (ex. /tmp/prefix); the generated file's full path is then passed in as a command line argument (ex. /tmp/prefix{unique string})
```

Comment:
```suggestion
    -f, --arg-file          Similar to --url-arg but the command line argument is a file containing new line separated values
```

Reply:
I'm not sure if this would make it clear that the argument file prefix needs to first be passed in and that the contents would be the URL arguments

Reply:
I think we should also say that the child process can do whatever it wants with the file (including deleting it)


(how you say it is a matter of taste and given that im not a maintainer of this repo, i will not give my suggestion :p )

Reply:
added a note to the end